tools/createTask.ts

```diff
@@ -0,0 +1,160 @@
+import gulp from 'gulp';
```

Comment:
I see this file will build all the packages with `yarn build:isolate`.

I want to have a command `yarn build:isolate Stack`.

And then will make a single gulu task `stack` with gulu pipe `gulu.pipe(source build).pipe(package json build).pipe(readme build);`.

So the `yarn build:isolate` will be:

```typescript
gulu.task('isolate', gulu.series('Stack', 'Queue', ...));
```

Reply:
now you can:
```
npx gulp isolate:stack
```

I've tried it, but it seems difficult to implement command like `yarn build:isolate Stack` with only use gulp.

Reply:
No problem, that's fine too. :)